src/UI/Components/InstanceProvider/InstanceProvider.tsx

```diff
@@ -5,10 +5,12 @@ import { RemoteDataView } from "@/UI/Components";
 import Canvas from "@/UI/Components/Diagram/Canvas";
 
 export const InstanceProvider: React.FC<{
+  label: string;
   services: ServiceModel[];
   mainServiceName: string;
   instanceId: string;
-}> = ({ services, mainServiceName, instanceId }) => {
+  editable: boolean;
```

Comment:
```suggestion
  editable?: boolean;
```
This should resolve your build issue :D